frontend/annotation_resolver.py

```diff
@@ -65,27 +69,27 @@ def resolve(self, annotation):
         if annotation[:4] == "List":
```

Comment:
Maybe using the AST nodes would be more elegant. 
@marcoeilers, what do you think?